src/python_testing/TC_RVCRUNM_2_2.py

```diff
@@ -0,0 +1,182 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import chip.clusters as Clusters
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main
+from mobly import asserts
+
+# This test requires several additional command line arguments.
+# Run the test with
+# --int-arg PIXIT_ENDPOINT:<endpoint> PIXIT.RVCRUNM.MODE_A:<mode id> PIXIT.RVCRUNM.MODE_B:<mode id>
+
+
+def error_enum_to_text(error_enum):
+    try:
+        return f'{Clusters.RvcRunMode.Enums.ModeTag(error_enum).name} 0x{error_enum:02x}'
+    except AttributeError:
+        if error_enum == 0:
```

Comment:
This is fine, but does modebase not get codegen'd? I can't find it in objects.py

Reply:
Unfortunately no. It's because ModeBase is not an actual cluster itself and does not have a cluster ID. The autogen tools do not yet understand this concept so we have had to write these structures and enums by hand for the rest of the SDK. I only realised that there were also python objects created by the autoget tools when writing these tests.